build/visible_to/BUILD

```diff
@@ -197,6 +197,7 @@ package_group(
         "//pkg/kubectl/cmd/auth",
         "//pkg/kubectl/cmd/resource",
         "//pkg/kubectl/cmd/set",
+        "//pkg/kubectl/explain",
```

Comment:
It's a little strange, but not horrible, that something not in kubectl/cmd wants to depend on cmd_testing.

The main purpose of these rules is to draw a line between kubectl and the rest, not small bits withing kubectl.

Reply:
And thanks for the fix!